test/bin/build_images.sh

```diff
@@ -24,20 +24,22 @@ if [ ! -d "${LOCAL_REPO}" ]; then
     error "Run ${SCRIPTDIR}/create_local_repo.sh before building images."
     exit 1
 fi
-release_info_rpm=$(find "${LOCAL_REPO}" -name 'microshift-release-info-*.rpm')
+release_info_rpm=$(find "${LOCAL_REPO}" -name 'microshift-release-info-*.rpm' -a -not -name '*_fake_*')
```

Comment:
We're going to have a few different RPMs now, especially when @copejon adds the "upgrade from HEAD to PR" test scenario. Maybe we can set up the RPM builds so they end up in different directories and different repos so it's easier to manipulate the files cleanly. Again, we can evolve this in another PR.

Reply:
I didn't notice any structure requirements for rpm repository. So maybe we could have one repository, but different subdirs?
Multiple repos would be just an overhead, if you feed them all to osbuilder, then you still need to specify versions to pick right one